ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -18,12 +18,53 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_centos
-- agent: jenkins
+- agent: kubernetes
   cron: ""
+  decorate: true
   interval: 24h
   labels:
     master: ci.openshift.redhat.com
   name: azure_build_base_image_rhel
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-azure-secret
```

Comment:
according to comments in github.com/openshift/release/pull/1774 i might need to add a suffix like `-base-image` to avoid surprises when pods of other jobs are configured with the same --secret-dir values. WDYT?

Reply:
If the secrets for those jobs were different then this might be a good idea.  For now, the secrets are identical and this is probably not required.

Reply:
copy that

Reply:
These are periodic jobs and each one should get its own namespace so this probably is not a problem for now. We may want to do something once we move them to run in PRs.